core/arch/x86/instr_create.h

```diff
@@ -1218,7 +1218,7 @@
 #define INSTR_CREATE_movshdup(dc, d, s) \
     instr_create_1dst_1src((dc), OP_movshdup, (d), (s))
 #define INSTR_CREATE_movddup(dc, d, s) instr_create_1dst_1src((dc), OP_movddup, (d), (s))
-#define INSTR_CREATE_pshufb(dc, d, s) instr_create_1dst_1src((dc), OP_pshufb, (d), (s))
+#define INSTR_CREATE_pshufb(dc, d, s) instr_create_1dst_2src((dc), OP_pshufb, (d), (s), (d))
```

Comment:
Please relocate this line to be with the `/* 1 destination, 2 sources: 1 explicit, 1 implicit */` group.

Reply:
Done